core/templates/services/ng-zone.service.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Downgraded NgZone service for use in AngularJS files.
+ */
+
+import { NgZone } from '@angular/core';
+import { downgradeInjectable } from '@angular/upgrade/static';
+
+angular.module('oppia').factory(
+  'NgZone', downgradeInjectable(NgZone));
```

Comment:
Where are we using this?

Reply:
Introduced as a fix for unstable zone due to ngAudio, it is no longer the case with howl. Going to remove this.